Libraries/polyfills/performance.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow strict
+ * @polyfill
+ */
+
+if (!global.performance) {
+  global.performance = {};
+}
+
+/**
+ * Returns a double, measured in milliseconds.
+ * https://developer.mozilla.org/en-US/docs/Web/API/Performance/now
+ */
+if (typeof global.performance.now !== 'function') {
+  global.performance.now = function() {
+    const performanceNow = global.nativePerformanceNow || Date.now;
```

Comment:
Is it possible for `global.nativeePerformanceNow` not to exist, after this change? It seems like you're adding this to all the executors, right?

Reply:
I think we should have a fallback for unofficial executors.